CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ### Bug Fixes
 
+1. [#5442](https://github.com/influxdata/chronograf/pull/5442): Updated table rendering styles to make table rows deterministic
```

Comment:
nit: i don't think 'deterministic' is the right word to use here. It has a pretty specific meaning in computer science: https://en.wikipedia.org/wiki/Deterministic_algorithm. if our programs aren't deterministic, we'd be completely and totally screwed. I think 'consistent' is might be better in context